assignment.md

```diff
@@ -12,15 +12,106 @@ The Kotlin assignment has three components
 - Create a working exmaple withing 'Let'
 - Create a working exmaple withing 'Run'  (Optional to do = Bonus!!)
 - Create a working exmaple withing 'Also'  (Optional to do = Bonus!!)
+- ---------------------------
+
+fun main(){
+###//          --->>>Create a working exmaple withing 'Let'<<<---
+val x = listOf(1,2,3).last().let { it * it }
+println(x)
+
+###    //          --->>>Create a working exmaple withing 'Run' (Optional to do = Bonus!!)<<<---
+    val r = listOf(1,2,3).last().run { 5 }
+    println(r)
+
+###    //          --->>>Create a working exmaple withing 'Also' (Optional to do = Bonus!!)<<<---
+    val ma = mutableListOf(1,2,3,4)
+    println(ma)
+        ma.add(5)
+    println(ma)
+        ma.also { it }
```

Comment:
wow, you did a great job but what do you think can you add an element to the "ma" list using also?